ov/server_hardware.go

```diff
@@ -620,19 +648,29 @@ func (c *OVClient) Patch(id string, operation []PatchData) error {
 	// refresh login
 	c.RefreshLogin()
 	c.SetAuthHeaderOptions(c.GetAuthHeaderMap())
+
+	t = t.NewProfileTask(c)
```

Comment:
is this necessary here? because we are dealing with server hardware

Reply:
Yes, its required for the resources which returns task uri to check task Status is complete or not.